src/cli/cli.go

```diff
@@ -147,7 +148,10 @@ func BuildCommandTree(cmd *cobra.Command) {
 // Handle builds and executes the cli processor.
 func Handle() {
 	//nolint:forbidigo
-	ctx := config.Seed(context.Background())
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(time.Hour*24*10))
```

Comment:
Nit: Not blocking here - but `context.WithTimeout(context.Background(), time.Hour*24*10)` is equivalent and easier to parse (do check if I'm correct though)

Reply:
not equivalent: Deadline takes in a time.Time, not a time.Duration (that's what I initially tried).

Reply:
@ryanfkeepers - to clarify - `WithTimeout` should take a duration.

Reply:
whoops, missed that, thanks